src/v2/mod.rs

```diff
@@ -1593,6 +1594,16 @@ impl Client {
         })
     }
 
+    /// Start a dry-run sequence that can be used to simulate a series of
+    /// transactions and other operations on the node.
+    ///
+    /// Before invoking any other operations on the [`dry_run::DryRun`] object,
+    /// the state must be loaded by calling
+    /// [`dry_run::DryRun::load_block_state`].
```

Comment:
I really don't like this sort of API design since it forces by far the most common use to be more verbose just to be slightly more general. This could easily just be different errors returned by the function.

Reply:
Ok. I will change this to `begin_dry_run` and add a new `dry_run` endpoint that loads a block.